requests/conda-smithy.yml

```diff
@@ -0,0 +1,3 @@
+action: broken
+packages:
+- noarch/conda-smithy-3.31.0-pyhd8ed1ab_1.conda
```

Comment:
> Not seeing where that changed happened

I'm guessing @beckermr did it in the UI, because `_1` followed `_0` pretty much [immediately](https://github.com/conda-forge/conda-smithy-feedstock/commits/67fc59b5ee04f64a7c80e1d5eaf8bf965bfffb55/) (~3min).

Reply:
Yep that is what it looks like

Mainly mentioning in case others are confused by the inclusion of `_1` here and no `_0`

Reply:
Yes sorry. Did it in the ui.